doc/RecordingControl.xml

```diff
@@ -2221,6 +2221,17 @@ secfrac = "." 1*6DIGIT</programlisting>
                 <para>Integer</para>
               </entry>
             </row>
+            <row>
+              <entry>
+                <para><literal>SHA256</literal></para>
+              </entry>
+              <entry>
+                <para>Hash of the uploaded segment.</para>
```

Comment:
I saw a PR#400 about full segment encryption. And this Hash of the segment will be stored in segment object metadata. Is this Hash calculated on encrypted segment or before segment encryption?